babs/cli.py

```diff
@@ -811,7 +812,8 @@ def _parse_merge():
         ),
         default=os.getcwd(),
     )
-    parser.add_argument(
+    dev_args = parser.add_argument_group('Developer arguments')
```

Comment:
As long as the developer argument group is clearly marked as "users shouldn't use this", I think it's fine to render it in the docs. Maybe add a description to the developer argument group like this:

```suggestion
    dev_args = parser.add_argument_group('Developer arguments', description='Parameters for developers. Users should not use these.')
```